src/common/logSubscriptionStatus.ts

```diff
@@ -0,0 +1,20 @@
+import { logErrorReturnLogId, logInfoReturnLogId } from "@/logger/logger";
+
+export const logSubscriptionStatus = async (
+    status: "TIMED_OUT" | "CHANNEL_ERROR" | "CLOSED" | "SUBSCRIBED",
+    err: Error | undefined,
+    tableName: string
+): Promise<boolean> => {
+    if (status === "TIMED_OUT") {
+        await logErrorReturnLogId(`Channel Timed Out: Subscribe to ${tableName} table`, err);
+        return true;
```

Comment:
What does this function return? Can we make that clear in the function name?

Reply:
Ok I've renamed to checkAndLogSubscriptionStatus because I think that's easier to read than logSubscriptionStatusAndReturnError or something. so have flipped the booleans, so it returns false if there's an error and true if there isn't.